src/runtime/client-config-manager-test.js

```diff
@@ -184,4 +184,30 @@ describes.realWin('ClientConfigManager', {}, () => {
     const clientConfig = await clientConfigManager.fetchClientConfig();
     expect(clientConfig.paySwgVersion).to.equal('2');
   });
+
+  it('getClientConfig should have useUpdatedOfferFlows after fetch', async () => {
+    const expectedUrl =
+      '$frontend$/swg/_/api/v1/publication/pubId/clientconfiguration';
+    fetcherMock
+      .expects('fetchCredentialedJson')
+      .withExactArgs(expectedUrl)
+      .returns(Promise.resolve({useUpdatedOfferFlows: true}))
```

Comment:
Instead of `.returns(Promise.resolve(` could you use `.resolves(` ?

Reply:
Thanks for the tip, updated.